lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/IMRUCloseTaskTest.cs

```diff
@@ -60,22 +62,15 @@ public void TestTaskCloseOnLocalRuntime()
             const int numTasks = 4;
             var testFolder = DefaultRuntimeFolder + TestId;
             TestBroadCastAndReduce(false, numTasks, chunkSize, dims, iterations, mapperMemory, updateTaskMemory, testFolder);
-            string[] lines = ReadLogFile(DriverStdout, "driver", testFolder);
-            var failedCount = GetMessageCount(lines, FailTaskMessage);
+            string[] lines = ReadLogFile(DriverStdout, "driver", testFolder, 120);
```

Comment:
Have we seen any case in which close event is _not_ received within 200 iterations? If something is so wrong that we don't get close event within 200 iterations, we probably won't get it in sleep case anyways (and we need to investigate what has gone wrong anyways). I'd say let's not overcomplicate things.